comparators/dirty_word.js

```diff
@@ -3,24 +3,32 @@
 module.exports = dirty_word;
 
 /**
-* Identify deletion of a city.
+* Identify potentially "dirty" words in any of the name tags.
 * @param {object} newVersion Features new version in GeoJSON.
 * @param {object} oldVersion Features old version in GeoJSON.
 * @param {Function} callback called with (error, result).
 * @returns {undefined} calls callback.
 */
 function dirty_word(newVersion, oldVersion, callback) {
 
-  var cfVersion = 1;
+  var cfVersion = 2;
 
   if (!newVersion || !newVersion.properties) {
     return callback(null, {});
   }
-  var dirtyWords = ['shit', 'dirty', 'fuck'];
+  var dirtyWords = ['shit', 'fuck', 'gangbang', 'gang-bang', 'pokemon', 'pokémon'];
```

Comment:
@bkowshik yes, it should be fine -- have modified the test fixture to test for this, though, and it looks 👍